main.go

```diff
@@ -104,21 +97,26 @@ func main() {
 
 	setupLog.Info("Build info",
 		"operatorVersion", OperatorVersion,
-		"mcadVersion", McadVersion,
-		"instaScaleVersion", InstaScaleVersion,
+		"appwrapperVersion", AppWrapperVersion,
 		"date", BuildDate,
 	)
 
 	ctx := ctrl.SetupSignalHandler()
 
+	namespace := namespaceOrDie()
 	cfg := &config.CodeFlareOperatorConfiguration{
 		ClientConnection: &config.ClientConnection{
-			QPS:   pointer.Float32(50),
-			Burst: pointer.Int32(100),
+			QPS:   ptr.To(float32(50)),
+			Burst: ptr.To(int32(100)),
+		},
+		AppWrapper: &config.AppWrapperConfiguration{
+			Enabled: ptr.To(true),
```

Comment:
This should be disabled by default to start with.

Reply:
Sure I can do that.  It might also allow us to merge this PR without waiting on SDK changes to enable the new AppWrappers (we'd need to be sure all the old MCAD code is disabled by default, but I think that has mostly been done already).

We should have a followup PR that adds e2e testing (another workflow?) in both this repo and the SDK that enables the AppWrapper controller though.

Reply:
Yes exactly, that makes things easier w.r.t. the SDK.

Reply:
Done.  I'll do a trial PR to the SDK that just makes the change to this branch of the operator but does the minimal other changes.